librosa/core/spectrum.py

```diff
@@ -999,8 +999,7 @@ def reassigned_spectrogram(
     ...     1e-6 * np.random.randn(2*sr)
     >>> freqs, times, mags = librosa.reassigned_spectrogram(y=y, sr=sr,
     ...                                                     n_fft=n_fft)
-    >>> mags_db = librosa.power_to_db(mags, ref=np.max)
-
+    >>> mags_db = librosa.amplitude_to_db(mags, amin=amin)
```

Comment:
I had to resolve a merge conflict here, but it looks like you've also changed the behavior of the db conversion.  Could you switch that back to `ref=np.max` instead of `amin=amin`?

Reply:
That was unintentional indeed. I had literally just copy/pasted the different function name, but my mistake was to branch off of `master` instead of `main` out of habit. Should be fixed by rebasing now.